manifests/e/Explorer/Patcher/22000.348.39.1/Explorer.Patcher.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Explorer.Patcher
+PackageVersion: 22000.348.39.1

```

Comment:
```
PackageVersion: 22000.348.39.1
```
cc @valinet: could you please add DisplayVersion registry key to your installer? At the moment, WinGet will keep upgrading this software because it has no version.